utils/useSwrvState.js

```diff
@@ -0,0 +1,67 @@
+import { ref, watchEffect } from '@vue/composition-api'
```

Comment:
Are we using `swrvState` anywhere for ktable?

Reply:
Yes, we are using swrv to handle revalidation with the fetcher.